wm-system/src/test/kotlin/com/ajaxproject/warehouse/controller/nats/product/UpdateProductNatsControllerTest.kt

```diff
@@ -22,17 +23,19 @@ class UpdateProductNatsControllerTest {
     lateinit var connection: Connection
 
     @Autowired
-    lateinit var productRepository: ProductRepository
+    lateinit var productRepository: ProductServiceInPort
```

Comment:
variable is `productRepository` but type `*Service*`. Please, rename the variable according to the type